TypeScript Error TS4090 – Conflicting definitions for 'node'

The package has a dependency on `@types/node`. Using this package in a TypeScript project that also has a dependency on `@types/node` causes this error:

```
error TS4090: Conflicting definitions for 'node' found at '<project_path>/node_modules/@types/node/ts3.2/index.d.ts' and '<project_path>/node_modules/@hubspot/api-client/node_modules/@types/node/index.d.ts'. Consider installing a specific version of this library to resolve the conflict.
```

I believe the accepted best practice for this is to define the `@types/*` packages as `devDependencies` and/or `peerDependencies` to prevent them from being installed in the receiving project's node_modules and allowing the user's project to determine the type definition to use.

--- a/package.json
+++ b/package.json
@@ -18,16 +18,16 @@
   "author": "HubSpot",
   "license": "ISC",
   "dependencies": {
-    "@types/bluebird": "^3.5.29",
-    "@types/lodash": "^4.14.149",
-    "@types/node": "^12.12.21",
-    "@types/request": "^2.48.4",
     "bluebird": "^3.7.2",
     "bottleneck": "^2.19.5",
     "lodash": "^4.17.19",
     "request": "^2.88.0"
   },
   "devDependencies": {
+    "@types/bluebird": "^3.5.29",
+    "@types/lodash": "^4.14.149",
+    "@types/node": "^12.12.21",
+    "@types/request": "^2.48.4",
     "eslint": "^6.5.1",
     "eslint-config-prettier": "^6.4.0",
     "eslint-config-standard": "14.1.0",

